optimum_energy_rubocop.yml

```diff
@@ -73,6 +88,39 @@ Layout/SpaceBeforeFirstArg:
 Layout/SpaceInsideStringInterpolation:
   EnforcedStyle: space
 
+###########
+# METRICS #
+###########
+
+Metrics/AbcSize:
+  Enabled: false
```

Comment:
This seems like a useful rule. Why as it disabled?

Reply:
Ditto to the answer before.